app/notes/notes.service.js

```diff
@@ -1,69 +1,75 @@
 (function() {
-  angular.module('meganote.notes')
+  angular
+    .module('meganote.notes')
     .service('NotesService', NotesService);
 
   NotesService.$inject = ['$http'];
```

Comment:
The constant LocalHost needs to be injected if it is going to be used in this service.